### PR TITLE
Add ctl-shift-i to indent binding

### PR DESCRIPTION
**Pull request by radamant**
I use `ctrl-i` for tab, so this binding works really well for me as the "smart" mode of `ctl-i`.

### CODE REVIEW

**Comment by aaronjensen** [comment]
how is that not shift tab?

**Comment by radamant** [comment]
Oooh boy. This is why I really don't like working on key bindings. Don't have an answer here. `shift-ctrl-i` works well for me. If it doesn't work for other folks, then kill this PR.

**Comment by aaronjensen** [comment]
what do you use as shift-tab? I'm just curious, i've never used ctrl-i as tab, you wanted that, but then you want ctrl-shift-i to be something other than shift-tab. ctrl-shift-i doesn't actually work as shift tab in the terminal.. so i dunno what it's meant to be

**Comment by radamant** [comment]
In emacs we use a plugin that makes tab widths into a ring, you can just cycle through by pressing tab/ctrl-i, it also does auto-indent if in a mode where it can determine indentation (like c-mode).

I almost never de-indent, I almost always select and re-auto-indent (which coincidentally is tab/ctrl-i magic).

**Comment by radamant** [comment]
Maybe `ctrl-i` would be good as auto-indent...

**Comment by aaronjensen** [comment]
is ctrl-= too out of reach? it's not the most convenient for me either, but i like the mnemonic of inside. i'm ok w/ ctrl-i as autoindent as long as we remove ctrl-=

**Comment by radamant** [comment]
Done and done. This now resolves #34 .

**Comment by aaronjensen** [comment]
need to move the other ctrl-i to ctrl-o and the ctrl-o to ctrl-shift-o

**Comment by aaronjensen** [comment]
oh and please update the readme w/ changes

**Comment by radamant** [comment]
I'll see if I can find time for this later, but it'll probably have to be tomorrow.

**Comment by radamant** [comment]
@aaronjensen anything else?

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Key                 | Command
                     | **Selecting Text**
 &#x2318;I           | Expand Region (word, inside parens, etc.)
 &#x21E7;&#x2318;I   | Shrink Region (word, inside parens, etc.)
-&#x2303;I (         | Select inside parentheses (works for `{[<'"t`)
+&#x2303;o (         | Select inside parentheses (works for `{[<'"t`)
```
Comment (aaronjensen) [inline README.md]: i've been using upper case letters regardless and the shift unicode char for shift.

**Comment by aaronjensen** [comment]
cool, thanks. please update the key binding for indent in the readme and add the shift arrow to the outside binding then :shipit:!

**Pull request merged**